libr/include/r_util/r_str.h

```diff
@@ -21,8 +21,17 @@ typedef enum {
 	R_STRING_ENC_GUESS = 'g',
 } RStrEnc;
 
+typedef struct {
+	char *str;
+	char buf[64]; // 20s vs 11s - 2x faster if string fits in buffer
```

Comment:
this 64 is constant. having a custom allocator or pool may require extra work in this data structure that im not sure if its worth, it depends on the use cases. comments here?